test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,28 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertHasNull(Object... objects) {assertTrue(Utils.isAnyNull(objects));}
+
+    private void assertNoNull(Object... objects) {assertFalse(Utils.isAnyNull(objects));}
+    @Test
+    public void isAnyNull() {
+        //empty list
+        assertNoNull();
+
+        //only one object
+        assertHasNull((Object) null);
+        assertNoNull(1);
+        assertNoNull("");
+        assertNoNull("abc");
+
+        //contains no null object
+        assertNoNull("abc","a");
+        assertNoNull(1,Integer.valueOf(1));
```

Comment:
Good consistency within the code, however note that our coding standard requires space after the comma.